### PR TITLE
Catch up with woothee v1.10.0

### PR DESCRIPTION
**Pull request by tell-k**
@okonomi 

I've catched up with woothee v1.9.0 and v1.10.0.
Please feel free to merge it. Thx.

### CODE REVIEW

File: src/AgentCategory/Browser/SafariChrome.php [hunk 1]
```diff
@@ -19,10 +19,11 @@ public static function challenge($ua, &$result)
 
         $version = DataSet::VALUE_UNKNOWN;
 
-        $epos = strpos($ua, 'Edge');
+        $epos1 = strpos($ua, 'Edge');
+        $epos2 = strpos($ua, 'Edg');
 
-        if ($epos > -1) {
-            if (preg_match('#Edge/([.0-9]+)#', $ua, $matches)) {
+        if ($epos1 > -1 || $epos2 > -1 ) {
```
Comment (okonomi) [inline src/AgentCategory/Browser/SafariChrome.php]: If it checks that it starts with `Edg`,` Edge` check may not be necessary.

File: src/AgentCategory/Os/Linux.php [hunk 2]
```diff
@@ -17,8 +17,9 @@ public static function challenge($ua, &$result)
         $version = null;
 
         if (strpos($ua, 'Android') !== false) {
-            $data = DataSet::get('Android');
-            if (preg_match('/Android[- ](\\d+\\.\\d+(?:\\.\\d+)?)/', $ua, $matches)) {
+          $data = DataSet::get('Android');
```
Comment (okonomi) [inline src/AgentCategory/Os/Linux.php]: Please fix indent.

**Comment by tell-k** [comment]
@okonomi Thank you for your review. I've fixed where you pointed it out. Please review this again. :)

**Comment by okonomi** [comment]
@tell-k Thank you for your contribution!

**Pull request merged**